Classpath loading on Windows via gitbash for CobiGen CLI is not working.

### Expected behavior

As a devonfw ide user on Windows, I want to load the CobiGen CLI classes and its dependencies dynamically.
The current classpath in the cobigen start script (executed through git bash) does not include the jars of the "software/cobigen-cli/lib/" folder correctly.

There seems to be an issue with the regex * expression on git bash in Windows.

### Actual behavior

Following error occurs when CobiGen is run:
`Error: Could not find or load main class com.devonfw.cobigen.cli.CobiGenCLI `
`Caused by: java.lang.ClassNotFoundException: com.devonfw.cobigen.cli.CobiGenCLI`

### Steps to reproduce (bug)

1. Install a fresh devonfw ide 2021.04.002 on Windows
2. Open a devon cmd using your context menu inside your project
3. Run: "devon cobigen" to install CobiGen CLI
4. Run "devon cobigen generate"

### Possible fix (Thanks to Jan-Gerrit):
Parsing of the regex expression through a printf
Needs to be adjusted in "software/cobigen-cli/bin/cobigen" file
`cp "$(printf %s: $LIB_DIR/*.jar)"`

### Related Issues:
https://github.com/devonfw/ide/issues/598

### Affected version:

* defonfw ide 2021.04.002
*

--- a/cobigen-cli/cli/src/main/assembly/resources/bin/cg
+++ b/cobigen-cli/cli/src/main/assembly/resources/bin/cg
@@ -1,7 +1,7 @@
 #!/bin/bash
 LIB_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../lib" && pwd )"
 if [ -z "$1" ]; then
-    java -cp "$LIB_DIR/*" com.devonfw.cobigen.cli.CobiGenCLI -help
+    java -cp "$(printf %s: $LIB_DIR/*.jar)" -help
 else
-    java -cp "$LIB_DIR/*" com.devonfw.cobigen.cli.CobiGenCLI "$@"
+    java -cp "$(printf %s: $LIB_DIR/*.jar)" com.devonfw.cobigen.cli.CobiGenCLI "$@"
 fi

--- a/cobigen-cli/cli/src/main/assembly/resources/bin/cg
+++ b/cobigen-cli/cli/src/main/assembly/resources/bin/cg
@@ -1,7 +1,10 @@
 #!/bin/bash
+# -cp "$LIB_DIR/*" is not working, please don't change this back!
+# Check: https://github.com/devonfw/cobigen/issues/1387
+# and https://stackoverflow.com/a/219801
 LIB_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../lib" && pwd )"
 if [ -z "$1" ]; then
-    java -cp "$(printf %s: $LIB_DIR/*.jar)" -help
+    java -cp "$(printf %s: $LIB_DIR/*.jar)" com.devonfw.cobigen.cli.CobiGenCLI -help
 else
     java -cp "$(printf %s: $LIB_DIR/*.jar)" com.devonfw.cobigen.cli.CobiGenCLI "$@"
 fi

